### PR TITLE
Adds Huggingface info to the examples README

### PR DESCRIPTION
**Pull request by marcvanzee**
# What does this PR do?

Fixes # (issue)

## Checklist
- [x] This PR fixes a minor issue (e.g.: typo or small bug) or improves the docs (you can dismiss the other
      checks if that's the case).
- [ ] This change is discussed in a Github issue/
      [discussion](https://github.com/google/flax/discussions) (please add a
      link).
- [ ] The documentation and docstrings adhere to the
      [documentation guidelines](https://github.com/google/flax/blob/main/docs/README.md#how-to-write-code-documentation).
- [ ] This change includes necessary high-coverage tests.
      (No quality testing = no merge!)

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -70,7 +70,13 @@ Graph modeling
 The following code bases use Flax and provide training frameworks and a wealth
 of examples, in many cases with pre-trained weights:
 
-- https://github.com/google-research/scenic: *Scenic* is a codebase/library
+- [HuggingFace Transformers](https://github.com/huggingface/transformers) is a library for bulding, training, and deploying
```
Comment (andsteing) [inline examples/README.md:73]: "building"

File: examples/README.md [hunk 2]
```diff
@@ -70,7 +70,13 @@ Graph modeling
...
+  state of the art models powered by the reference open source in machine learning.
```
Comment (andsteing) [inline examples/README.md:74]: "powered by the reference open source in machine learning" ... this sentence sounds a bit weird; also I do agree that HF is a very important and widely used library with tons of greatness, but I wouldn't go as far as calling it the "reference OSS in ML"

**Review by andsteing — APPROVED** [review]
Thanks for adding this. This is very useful!

See two comments about wording, but I'll already approve to save you another cycle.

File: examples/README.md (hunk 2 above)
Comment (marcvanzee) [reply examples/README.md:74]: Ok I agree, I actually copied this from their website, I now toned it down a bit 😃 .

**Pull request merged**